ch08.adoc

```diff
@@ -68,3 +68,467 @@ variables:
+
+The definition of the interpolation method, however it is specified, may include instructions to treat groups of physically related coordinates simultaneously, if such tie points are present. For example, there are cases where longitudes cannot be interpolated without considering the corresponding latitudes. It is up to the interpolation description to describe how such coordinates are to be identified (e.g. it may be that such tie point variables require particular units or standard names).
+
+An interpolation method may require __interpolation coefficient variables__ that provide values for interpolation equation terms that are not satisfied by the tie points. Such terms in the interpolation equations are associated with interpolation coefficient variables by the **`interpolation_coefficients`** attribute that takes a string value, the string being comprised of blank-separated elements of the form `"term: variable"`, where `term` is a case-insensitive keyword that defines one of the terms in the interpolation method's definition, and `variable` is the name of the interpolation coefficient variable that contains the values for that term. The order of elements is not significant. A term that is omitted from the **`interpolation_coefficients`** attribute should be assumed to be zero.
+
+The interpolation variable attribute **`interpolation_configuration`** may be used to configure the interpolation process. This attribute names other __interpolation configuration variables__ that contain parameters needed to correctly configure the interpolation process. The **`interpolation_configuration`** attribute takes a string value, the string being comprised of blank-separated elements of the form `"item: variable"`, where `item` is a case-insensitive keyword that identifies a configuration item defined in the interpolations method's definition, and `variable` is the name of the interpolation configuration variable that contains the values for that item. The order of elements is not significant.
+
+The **`interpolation_coefficient`** and **`interpolation_configuration`** attributes may only be provided if allowed by the definition of the interpolation method.
+
+The variables named by the **`interpolation_coefficients`** and **`interpolation_configuration`** attributes must either be scalar, or else their dimensions may include, for each interpolation dimension, either the corresponding tie point interpolation dimension or the corresponding interpolation zone dimension, but not both, and may include any of the non-interpolation dimensions.
```

Comment:
Why would an interpolation coefficient/configuration variable vary along a non-interpolation dimension? It clearly can't vary along any non-interpolation variable that is _not_ spanned by the tie point variables, and I don't know what it means for it to vary along any that are. Could we instead pin this down a bit more:

_The dimensions, if any, of the variables named by the **`interpolation_coefficients`** and **`interpolation_configuration`** attributes must correspond to any subset of the interpolation dimensions. Each dimension must be either the tie point interpolation dimension or interpolation zone dimension that corresponds to an interpolation dimension, and may not include any of the non-interpolation dimensions._

Reply:
Hi @davidhassell

I believe it is correct that  interpolation coefficient/configuration can vary along non-interpolation dimension in the same way as a tie point coordinate values can. Take our Example 8.4. One-dimensional tie point interpolation of two-dimensional domain. If this example had a quadratic interpolation method, then the interpolation coefficients would be stored once for each index of the non-interpolation dimension and would likely be different for each index value. Recall that the the interpolation method is being applied multiple times along the interpolation dimension, once for each index of the non-interpolation dimension.